### DISCUSSION TITLE
Will there be a Sameboy Advance with link cable support?

### DISCUSSION THREAD

**Discussion by HelpMeGetANamePlease** [LIJI32/SameBoy#529] [General]
I really like the Sameboy core in Retroarch mainly due to being able to play Penguin Wars with link cable emulation. GBA also has many great multiplayer games. But I can't play them on my phone with more than 2 players. I'm just asking if there is a possibility for a Sameboy Advance with upto 4-player link cable support.

**Comment by LIJI32** [comment]
The GBA is a completely different platform, and other than partial possible reuse of the audio subsystem, "SameBoy Advance" would be a completely new emulator. It would take years for it to reach the emulation accuracy of current top-2 GBA emulators, [mGBA](https://github.com/mgba-emu/mgba) and [NanoBoyAdvance](https://github.com/nba-emu/NanoBoyAdvance), and it's not a journey I'm willing to take. It would be much time effective to add libretro link cable support for one of these two existing emulators.
